osm2gtfs/creators/trips_creator.py

```diff
@@ -317,9 +323,12 @@ def _load_scheduled_stops(self, schedule, itinerary, service):
         stops = []
         for trip in schedule['lines'][itinerary.route_id]:
             trip_services = trip["services"]
+            if "via" not in trip:
+                trip["via"] = None
             if (trip[
                     "from"] == itinerary.fr and trip[
-                        "to"] == itinerary.to and service in trip_services):
+                        "to"] == itinerary.to and trip[
+                        "via"] == itinerary.via and service in trip_services):
```

Comment:
It was already like this, but breaking open the accessor like this looks like bad style. Can we keep `trip["via"]` together without line breaks?